composer.json

```diff
@@ -17,15 +17,15 @@
     ],
     "require": {
         "php": ">=5.6",
-        "symfony/monolog-bridge": "~2.7|~3.3|~4.0",
-        "symfony/dependency-injection": "~2.7|~3.4.10|^4.0.10",
-        "symfony/config": "~2.7|~3.3|~4.0",
-        "symfony/http-kernel": "~2.7|~3.3|~4.0",
+        "symfony/monolog-bridge": "~3.3|~4.0",
```

Comment:
should we change it to `~3.4` too as 3.3 is EOLed (2.8 still has security support for a few months, but 3.3 does not)

Reply:
I thought the same thing! But I did not want to bump for the sake of bumping.
But I would go for too

Reply:
Let's bump!